src/contracts/StakedTokenV3.sol

```diff
@@ -517,7 +531,7 @@ contract StakedTokenV3 is
       } else {
         stakersCooldowns[from].amount =
           stakersCooldowns[from].amount -
-          amountToRedeem.toUint184();
+          amountToRedeem.toUint104();
```

Comment:
the cast was wrong (as it's already casted toUint104 5 lines above)

generally safeCast seems unnecessary here, given that a safeCast is performed a few lines above